R/lucid.R

```diff
@@ -1,8 +1,24 @@
+lucidClientForAccount <- function(account) {
+  authInfo <- account
 
-# return a list of functions that can be used to interact with lucid
-lucidClient <- function(service, authInfo) {
-  service <- parseHttpUrl(service)
+  # determine appropriate server information for account
+  if (account$server == cloudServerInfo()$name) {
```

Comment:
Does this check allow for staging/prod testing?

Reply:
It does because you can change the `rsconnect.shinyapps_url` options which targets a new URL - but doesn't influence the name.